pyproject.toml

```
@@ -1,6 +1,6 @@
 [project]
 name = "rbyte"
-version = "0.28.1"
+version = "0.28.2"
```

```suggestion
version = "0.29.0"
```

functionality change